nixos/modules/services/monitoring/grafana.nix

```diff
@@ -366,10 +430,15 @@ in {
         User = "grafana";
       };
       preStart = ''
+        mkdir -p ${cfg.dataDir}/provisioning/{dashboards,datasources}
+        cat ${datasourceConfig} > ${cfg.dataDir}/provisioning/datasources/datasource_conf.yml
+        cat ${dashboardConfig} > ${cfg.dataDir}/provisioning/dashboards/dashboard_conf.yml
+        cat ${dashboardFile} > ${cfg.dataDir}/provisioning/dashboards/dashboard.json
```

Comment:
Useless use of cat, you can just `ln -s` these. Also, doesn't this break backwards compatibility for people that had these files specified already?